workflows/taxprofiler.nf

```diff
@@ -209,6 +210,12 @@ workflow TAXPROFILER {
     PROFILING ( ch_reads_runmerged, DB_CHECK.out.dbs )
     ch_versions = ch_versions.mix( PROFILING.out.versions )
 
+    /*
+        SUBWORKFLOW: VISUALIZATION
+    */
+    VISUALIZATION ( PROFILING.out.profiles )
```

Comment:
I would like to make this optional, so maybe wrap it with an if statement and evaluating a new  `params.run_visualisation_krona` or something.

Reply:
See 89d4d1a